editor/editor_file_system.cpp

```diff
@@ -2089,6 +2089,60 @@ void EditorFileSystem::reimport_file_with_custom_parameters(const String &p_file
 	_reimport_file(p_file, &p_custom_params, p_importer);
 }
 
+void EditorFileSystem::overwrite_file(const String &p_file, Ref<Resource> p_resource) {
+	EditorFileSystemDirectory *fs = nullptr;
+	int cpos = -1;
+	bool found = _find_file(p_file, &fs, cpos);
+	ERR_FAIL_COND_MSG(!found, "Can't find file '" + p_file + "'.");
+
+	ResourceUID::ID uid = ResourceUID::INVALID_ID;
+
+	if (FileAccess::exists(p_file + ".import")) {
+		//use existing
+		Ref<ConfigFile> cf;
+		cf.instantiate();
+		Error err = cf->load(p_file + ".import");
+		if (err == OK) {
+			if (cf->has_section("remap")) {
+				if (cf->has_section_key("remap", "uid")) {
+					String uidt = cf->get_value("remap", "uid");
+					uid = ResourceUID::get_singleton()->text_to_id(uidt);
+				}
+			}
+		}
+	}
+
+	String base_path = ResourceFormatImporter::get_singleton()->get_import_base_path(p_file);
+	ResourceSaver::save(p_resource, base_path + ".scn"); //todo: get prefered save extension from somewhere
+
+	//update modified times, to avoid reimport
+	fs->files[cpos]->modified_time = FileAccess::get_modified_time(p_file);
+	fs->files[cpos]->import_modified_time = FileAccess::get_modified_time(p_file + ".import");
+	fs->files[cpos]->deps = _get_dependencies(p_file);
+	fs->files[cpos]->type = p_resource->get_class();
+	fs->files[cpos]->uid = uid;
+	fs->files[cpos]->import_valid = fs->files[cpos]->type == "TextFile" ? true : ResourceLoader::is_import_valid(p_file);
+
+	if (ResourceUID::get_singleton()->has_id(uid)) {
+		ResourceUID::get_singleton()->set_id(uid, p_file);
+	} else {
+		ResourceUID::get_singleton()->add_id(uid, p_file);
+	}
+
+	//if file is currently up, maybe the source it was loaded from changed, so import math must be updated for it
```

Comment:
Godot Engine comments are typically using `//` and are sentence case.

Reply:
This is just a copy-paste, the save logic is just copied and reduced from `reimport_file` to work as temporary solution.